modules/ROOT/pages/api-groups-modifying-groups.adoc

```diff
@@ -48,22 +48,13 @@ You might want to add API instances to an existing API Group to widen the busine
 
 To add API instances to an existing API Group:
 
-. From the left menu in *API Manager*, click *API Groups*:
-+
-.The screenshot illustrates the page that is displayed for API Groups.
-image:add-api-instance-modify-api-groups.png[Add API Instances to API Groups]
-+
-<1> API Group version
-<2> Instances of the API Group version
-<3> API Group
-<4> New version of the API Group
-<5> New instance of the API Group version
+. From the left menu in *API Manager*, click *API Groups*.
 . On the API Groups details page, expand the API Group to view its available versions.
-. Click the version of the API Group to which you want to add the API instance and click *Add Instance*.
+. Click *Add group instance* for the version of the API Group you want to add the API instance to.
 +
 The *Creating a new group instance* page is displayed.
-. Click *Add*.
-. If the API instance and the API Group belong to the same business group or organization, select the environment, API instance, and version that you want to add to the API Group.
+. Click *+Add*.
+. If the API instance and the API Group belong to the same business group or organization, select the API, API version, environment, and API version/instance that you want to add to the API Group.
```

Comment:
> select the API, API version, environment, and API version/instance

API version appears two times. I think it could be: `select the API, environment, and API version/instance`

Reply:
So there are actually 4 drop downs here and all four need to be selected.